agent/custom/bbc_action.py

```
@@ -431,6 +438,7 @@ def _setup_and_start_battle(self, team_config: str, run_count: int,
     def _handle_popups(self, messages: list, support_order_mismatch: bool, 
```

Ruff (0.15.10)

[warning] Too many return statements (7 > 6)

(PLR0911)